docs/D-Modularity/library-functions.md

```diff
@@ -139,14 +143,14 @@ The following program outputs a different set of 10 pseudo-random numbers with e
 ```c
 #include <stdlib.h>
 #include <stdio.h>
-#include <time.h> // prototype for time(NULL)
+#include <time.h> // ...prototype for time(NULL)...
```

Comment:
Throughout this file, please remove the `... ...` that you have added.  They aren't needed.